packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1812,13 +1839,11 @@ export default class Meeting extends StatelessWebexPlugin {
             mediaSettings,
             localShare,
             localStream
-          }).then((mediaResponse) => {
-            Promise.resolve({
-              join: joinResponse,
-              media: mediaResponse,
-              local: [localStream, localShare]
-            });
-          })))
+          }).then((mediaResponse) => Promise.resolve({
```

Comment:
No need to `Promise.resolve` since this is in a promise chain. Simply return the object:

```js
          }).then((mediaResponse) => ({
             join: joinResponse,
             media: mediaResponse,
             local: [localStream, localShare]
```